content/applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst

```diff
@@ -1,101 +1,271 @@
+and components on a monthly basis.
+
+The :guilabel:`Number of Columns` field is used to specify the quantity of the selected
+:guilabel:`Time Range` units shown on the |MPS| page. For example, if the :guilabel:`Time Range`
+field is set to :guilabel:`Monthly`, and `12` is entered in the :guilabel:`Number of Columns` field,
+the |MPS| shows one column for the next 12 months, starting with the current month.
+
+If the values of the :guilabel:`Time Range` or :guilabel:`Number of Columns` fields are altered,
+click :guilabel:`Save` again to save the changes.
+
+.. image:: use_mps/mps-setting.png
+   :align: center
+   :alt: The MPS setting in the Manufacturing app's settings.
+
+MPS UI
+======
+
+To open the |MPS|, navigate to :menuselection:`Manufacturing app --> Planning --> Master Production
+Schedule`. The |MPS| view appears as follows:
+
+.. image:: use_mps/mps.png
+   :align: center
+   :alt: The Master Production Schedule in the Manufacturing app.
+
+The grey column on the left side of the screen shows a section for every product added to the |MPS|,
+with each product section being broken down into smaller rows. The information shown in the rows
+depends on the filters selected in the :guilabel:`Search...` bar drop-down menu at the top of the
+page. The default categories that appear in the rows are:
+
+- :guilabel:`[Product] by [unit]` :icon:`fa-area-chart`: the forecasted stock quantity at the
+  beginning of each time period. :guilabel:`[Product]` and :icon:`fa-area-chart` are selectable
+  buttons which open the product's page, or the forecast report for the product, respectively.
+- :guilabel:`- Forecasted Demand`: the demand forecast, which is entered manually. This represents
+  an estimate of the demand for the product during each time period.
+- :guilabel:`- Indirect Demand Forecast`: while this is a default category, it **only** appears for
+  products that are components of other products. It represents the demand for the component from
+  existing MOs.
+- :guilabel:`+ Suggested Replenishment`: the quantity of the product that is suggested to be
+  replenished through |MOs| or |POs|. To the right of the category title is a
+  :guilabel:`Replenish` button, which is used to manually replenish the product, based on the
+  quantity suggested to be replenished.
+
+  .. figure:: use_mps/replenish-button.png
+     :align: center
+     :alt: The Replenish button on the "+ Suggested Replenishment" row.
+
+     The "Replenish" button on the "+ Suggested Replenishment" row.
+
+- :guilabel:`= Forecasted Stock`: the quantity of the product forecasted to be in stock at the end
+  of each time period, assuming that suggested replenishment numbers are fulfilled.
+
+Altogether, these default categories form an equation: `Forecasted Demand + Suggested Replenishment
+= Forecasted Stock`. In the case of components, the :guilabel:`Indirect Demand Forecast` is taken
+into account as well.
```

Comment:
Suggestion to use the [math directive](https://docutils.sourceforge.io/docs/ref/rst/mathematics.html) here for nice maths-like formatting :)
```suggestion
Altogether, these default categories form an equation:

.. math::
   \text{Forecasted Demand} + \text{Suggested Replenishment} = \text{Forecasted Stock}

In the case of components, the :guilabel:`Indirect Demand Forecast` is taken into account as well.
```